src/backend/parser/parse_clause.c

```diff
@@ -908,6 +909,13 @@ transformRangeTableFunc(ParseState *pstate, RangeTableFunc *rtf)
 										  tf, rtf->alias, is_lateral, true);
 }
 
+/* Wrapper function that calls the static transformRangeTableFunc function */
+ParseNamespaceItem *
+bbf_transformRangeTableFunc(ParseState *pstate, RangeTableFunc *rtf)
+{
+    return transformRangeTableFunc(pstate, rtf);
+}
```

Comment:
Thats not how we should expose functions in opensource. Is there no way we can expose it through transformFromClause?